proto/text.go

```diff
@@ -594,15 +594,15 @@ func writeString(w *textWriter, s string) error {
 
 func writeUnknownStruct(w *textWriter, data []byte) (err error) {
 	if !w.compact {
-		if _, err := fmt.Fprintf(w, "/* %d unknown bytes */\n", len(data)); err != nil {
+		if _, err := fmt.Fprintf(w, "# %d unknown bytes\n", len(data)); err != nil {
```

Comment:
While I prefer removing this, if you feel strongly that this is beneficial, feel free to keep it.